### PR TITLE
Add support for prefixing margin & padding logical properties

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: src/Prefixer.js [hunk 1]
```diff
@@ -109,6 +109,9 @@ export function prefixer (value, length) {
 			 	(WEBKIT+value+';') +
 			 	(value)
 			)
+		// (margin|padding)-inline-(start|end)
+		case 4095: case 3583: case 4068: case 2532:
+			return (WEBKIT+replace(value, '-inline', '')+';') + (value)
```
Comment (Andarist) [inline src/Prefixer.js]: alternatively we could use regexp like this:
```js
return (replace(value, /(.+)-inline(.+)/, WEBKIT+'$1$2')+';') + (value)
```

not sure what is preferred

Comment (thysultan) [reply src/Prefixer.js]: That works.

Comment (Andarist) [reply src/Prefixer.js]: that as in simple string (committed change) or the regexp one? :p

Comment (thysultan) [reply src/Prefixer.js]: regexp

Comment (Andarist) [reply src/Prefixer.js]: Ok, merging that version then 👍

**Pull request merged**